rtsp.go

```diff
@@ -110,12 +114,12 @@ func (rc *rtspCamera) clientReconnectBackgroundWorker(codecInfo videoCodec) {
 				badState = true
 			} else {
 				res, err := rc.client.Options(rc.u)
-				rc.logger.Debugf("Options reponse: %s, err: %s", res, err)
+				rc.logger.Debugf("Options response: %s, err: %s", res, err)
 				// Nick S:
 				// This error happens all the time on hardware we need to support & does not affect
 				// the performance of camera streaming. As a result, we ignore this error specifically
-				_, isErrClientInvalidState := err.(liberrors.ErrClientInvalidState)
-				if err != nil && !isErrClientInvalidState {
+				var errClientInvalidState *liberrors.ErrClientInvalidState
+				if err != nil && !errors.As(err, &errClientInvalidState) {
```

Comment:
linter suggests using `errors.As`

According to linter, it takes into account wrapped errors, and is better practice

Reply:
Can you test this on the physical IP camera to confirm that we trigger reconfigure for all errors other than `ErrClientInvalidState`?

Reply:
It would take me a good amount of time to find an physical RTSP cam to use + set it up or bother Sean P or you to do that. The pattern in these two lines is pretty common in Go 1.13+ and is the recommended approach to error type checking. Unless you're very adamant about doing this manual test, I would recommend we just go forward with this change 

https://archive.ph/HwjhY